plugins/europeana/record.js

```diff
@@ -257,6 +261,10 @@ export default (axiosOverrides) => {
 
         // Inject service definitions, e.g. for IIIF
         webResource.services = services.filter((service) => (webResource.svcsHasService || []).includes(service.about));
+
+        // Remove unused data
+        delete webResource.htmlAttributionSnippet;
+        delete webResource.textAttributionSnippet;
```

Comment:
I think it's going to be easier long term if we declare the fields we do want, then only select those. Instead of removing specific ones.

Reply:
Indeed, and done.